dataland-frontend/src/components/general/TheJoinCampaign.vue

```diff
@@ -0,0 +1,171 @@
+<template>
+  <section v-if="joinCampaignSection" class="joincampaign" role="region" aria-labelledby="joincampaign-heading">
+    <div class="joincampaign__wrapper">
+      <h2 id="joincampaign-heading" class="joincampaign__title">
+        {{ joinCampaignSection.text[0] }}
+        <span v-if="joinCampaignSection.text[1]"> {{ joinCampaignSection.text[1] }}</span
+        >{{ joinCampaignSection.text[2] }}
+      </h2>
+      <p class="joincampaign__paragraph" v-if="joinCampaignSection.text[3]">{{ joinCampaignSection.text[3] }}</p>
+      <email-button :sections="sections" />
```

Comment:
emailButton should be renamed to "EmailButton" to be in line with our naming convention

Reply:
done

Reply:
Somehow it is still "emailButton.vue" for me and in the remote github repo (see other comment with screenshot). Strange,